### PR TITLE
add 'GetPolicy' to sts ocm user role policy

### PR DESCRIPTION
**Pull request by pvasant**
### What type of PR is this?
Feature

### What this PR does / why we need it?
For ROSA UI

### Which Jira/Github issue(s) this PR fixes?
[SDA-5960](https://issues.redhat.com//browse/SDA-5960)

_Fixes #_

### Special notes for your reviewer:
Approved by BU

### Pre-checks (if applicable):
- [x] Tested latest changes against a cluster
- [ ] Included documentation changes with PR

### CODE REVIEW

**Comment by jewzaam** [comment]
/hold

waiting on PM approval in JIRA

**Review by arendej — APPROVED** [review]
approved

**Pull request merged**
